src/plugins/dashboard/public/application/actions/copy_to_dashboard_modal.tsx

```diff
@@ -76,11 +77,12 @@ export function CopyToDashboardModal({
         : `#${DashboardConstants.CREATE_NEW_DASHBOARD_URL}`;
 
     closeModal();
+
     stateTransfer.navigateToWithEmbeddablePackage('dashboards', {
       state,
       path,
     });
-  }, [dashboardOption, embeddable, selectedDashboard, stateTransfer, closeModal]);
+  }, [embeddable, dashboardOption, selectedDashboard, closeModal, stateTransfer]);
```

Comment:
Can you discard the changes to this file and `copy_to_dashboard_action`? It seems like none of these changes are actually relevant

Reply:
Done.